content/posts/introducing-slight.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Introducing SpiderLightning - A Cloud System Interface with WebAssembly"
+description: "Announcing slight"
+date: 2022-10-30 00:00:00 +0000 UTC
+authorname: "Jiaxiao (Joe) Zhou"
+author: "@jiaxiao_zhou"
+authorlink: "https://twitter.com/jiaxiao_zhou"
+image: "images/logos/twitter-card.png"
+tags: ["slight", "spiderlightning"]
+---
+
+At DeisLabs, we are researching and developing various WebAssembly tools and systems overtime. We have been working on a new project called SpiderLightning, which is a cloud system interface written in a Interface Definition Language (IDL) called [WIT](https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md). We have also experimented with a CLI called slight, which we recently released "0.2.0" version. In this blog post, we will introduce SpiderLightning and slight.
+
+## Authoring portable distributed applications is hard
+
+Imagine you're a developer working at a consumer-facing company A. You built a web app that runs on on-premises data centers and used open-source projects to build data stores and pipelines. At beginning, there are only a few hundreds of customers, so the web app worked well. However, as the company grows, the number of customers increases rapidly. The web app becomes slow and unstable. Now you're tasked to migrate from on-premises data centers to public cloud.
```

Comment:
```suggestion
Imagine you're a developer working at a consumer-facing company A. You built a web app that runs on on-premises data centers and used open-source projects to build data stores and pipelines. At the beginning, there are only a few hundreds of customers, so the web app worked well. However, as the company grows, the number of customers increases rapidly. The web app becomes slow and unstable. Now you're tasked to migrate from on-premises data centers to public cloud.
```

Reply:
I think we need to work on this paragraph a bit as it's the hook that will get folks to continue reading.

I'm not clear what building data stores and pipelines are. I mean, I can kind of imagine what is going on, but it isn't super clear to me. Consider something really simple the application does. Maybe it handling airline reservations or some other use case that is easily relatable and less tech focused.

Also consider cutting anything extra from the paragraph. One sentence that jumps out at me is "The web app becomes slow and unstable." I don't believe that slight / SL will magically make a poorly written / scaling application into a well written, linearly scalable application. Try to focus on the problem that you are solving, making the application more portable and have lower cost of running due to reduced resource requirements.

Reply:
The next paragraph is the great for describing the agnostic APIs. Lean more into that.